make-options.json

```diff
@@ -40,8 +40,6 @@
         "DownloadSecureFile",
         "ExtractFiles",
         "FtpUpload",
-        "Go",
```

Comment:
Once you check-in verify the package generated for releases/m131 to confirm the tasks were really excluded.

Reply:
Sure, I will check that